### DISCUSSION TITLE
Stay in the line....

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#75] [General]
Some boxes displayed not on a line with each others, see below:

![Screenshot 2023-05-24 at 20 15 53](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/b5ee2af4-3aee-4609-ba46-7115a1da00a0)

**Comment by TJ-BitCanna** [comment]
Thanks for noticing!

This is also part of the new design (a sneak preview you've found), where these issues indeed have to be tackled. 

I'll get back on this after internal discussion what we'll do time-wise regarding design.

**Comment by TJ-BitCanna** [comment]
This is indeed something we will need to check and revise.
